src/Protocol/ActivityPub/Processor.php

```diff
@@ -180,6 +180,38 @@ public static function updateItem($activity)
 		}
 
 		Item::update($item, ['uri' => $activity['id']]);
+
+		if ($activity['object_type'] == 'as:Event') {
+			$posts = Post::select(['event-id', 'uid'], ['uri' => $activity['id']]);
+			while ($post = DBA::fetch($posts)) {
+				if (empty($post['event-id'])) {
+					continue;
+				}
```

Comment:
Can't we add this condition to the query like this?
```suggestion
			$posts = Post::select(['event-id', 'uid'], ['uri' => $activity['id'], 'event-id' => true]);
			while ($post = DBA::fetch($posts)) {
```

Reply:
The `event-id` is an integer field. I don't think that this would work.

Reply:
The following query only returns rows with populated `event-id`:
```sql
SELECT `id`, `event-id` FROM `post-user` WHERE `event-id`;
```

Reply:
The query above is translated slightly different. Also we cannot guarantee that this will work with each SQL flavour. Oracle MySQL for example is getting more and more picky and less forgiving that MariaDB. So I'm really cautious to not produce something that could be a problem.

Reply:
Aren't we using this notation for boolean fields which are just short unsigned integers?

Reply:
Thing is: We don't know how other SQL variants are behaving. For this reason I changed the query.